.github/workflows/update.yml

```diff
@@ -11,8 +11,7 @@ jobs:
       - uses: actions/checkout@v2
       - uses: actions/setup-node@v2
       - run: npm ci
-      - run: npm install @octokit/webhooks-definitions@latest
-      - run: npm run update-known-events
+      - run: npm install -ED @octokit/webhooks-definitions@latest
```

Comment:
I'd prefer not to use abbreviations. They are great for typing commands in your terminal, but they are harder to read/understand to others and your future self. Can you use the full flag names instead?